alchemy/src/cloudflare/website.ts

```diff
@@ -353,21 +353,71 @@ async function runDevCommand(
     cwd?: string;
   },
 ) {
-  const persistFile = path.join(process.cwd(), ".alchemy", `${props.id}.pid`);
-  if (await exists(persistFile)) {
-    const pid = Number.parseInt(await fs.readFile(persistFile, "utf8"));
-    try {
-      // Actually kill the process if it's alive
-      process.kill(pid, "SIGTERM");
-    } catch {
-      // ignore
+  // Directory storing per-process PID files: .alchemy/pids/{id}-{pid}.pid
+  const pidDir = path.join(process.cwd(), ".alchemy", "pids");
+  const pidPrefix = `${props.id}-`;
```

Comment:
`props.id` is not globally unique if `id` is the Resource ID.

We could instead compute one from `resource.chain` (and remove any `/`, or store them as nested folders like we do for file system state store)